### PR TITLE
Script to add csv data to database

### PR DESCRIPTION
**Pull request by vviveksharma**
- This the script that reads the csv or excel sheet data provided which has details of SKU and its related products
## Description

- Created a script that reads the csv data provided
- Make a dictionary out of it with key values as SKU products and values as the list of related products
- Make a connection to dynamoDB ( relatedProducts )
- Pushed the data on it 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [x] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

Demo Video for the implementation 

https://progresssoftware.sharepoint.com/:v:/s/ChefCoreC/EQvmqzyCl0ZDiKJaxvQh47sB2DHgWp6NA5-ioVIjEWL3sA?e=BxXg0M

Screen Shot for the loaded values in the dataBase
Screenshot 2023-08-25 at 2 46 28 PM

### CODE REVIEW

**Comment by gangwarshivendra** [comment]
How and where this will be executed in production, Do we need to set up some packages for execution?

**Review by gangwarshivendra — APPROVED** [review]
Looks good to me. I believe this one will be executed periodically.

**Review by shanmugapriya-tr — APPROVED** [review]
LGTM

File: scripts/push_data_to_database.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            response_dict[j] = response
+        return response_dict
+    except Exception as e:
+        return f"An error occurred: {str(e)}"
+
+
+def push_to_database(filepath):
+    data = get_Columns(filepath)
+    session = create_session()
+    table_name = os.getenv('SKU_TABLE_NAME')
```
[the remark this answers is not in the document]
Comment (vviveksharma) [reply scripts/push_data_to_database.py]: Done updated the file path as that of env variable

**Review by vipin230 — APPROVED** [review]

**Review by vipin230 — APPROVED** [review]

**Review by shanmugapriya-tr — APPROVED** [review]

**Pull request merged**